en/docs/learn/provisioning-users-to-hubspot.md

```diff
@@ -0,0 +1,154 @@
+4. Click on **List** under **Claims** on the left pane and click **http://wso2.org/claims**.
+5. Click edit on &quot; **Telephone**&quot; and uncheck supported and required checkboxes.
+
+    ![telephone-claim](../assets/img/tutorials/telephone-claim.png)
+
+6. Then click edit on &quot; **Phone Numbers**&quot; and select supported and required checkboxes.
+
+    ![phone-number-claim](../assets/img/tutorials/phone-number-claim.png)
+
+#### Step 02: Configure HubSpot as the identity Provider
+
+1. On the Management Console, click **Add** under **Identity Providers**.
+2. In the form that appears, provide a name for your identity provider by filling in the **Identity Provider Name**. 
+You can use &quot;HubSpot.com&quot; as an example, but this can be any name you choose. See 
+[Configuring an Identity Provider](https://is.docs.wso2.com/en/latest/learn/adding-and-configuring-an-identity-provider/) 
+for information on registering and configuring an identity provider.
+3. Expand the **Outbound Provisioning Connectors** section followed by the **SCIM Provisioning Configuration** section.
+
+4. Check the **Enable** Checkbox and give the **Username** and **Password** as given below.
+5. The value of the User Endpoint should be the same as the SCIM receiving endpoint that we deployed in the MI.
+
+    ```
+        Username: admin
+        Password: admin
+        User Endpoint: https://localhost:8253/hubspot/contacts
+    ```
+
+    ![scim-provisioning-conf](../assets/img/tutorials/scim-provisioning-conf.png)
+
+1. Click Update and save the configuration
+
+#### Step 03: Configuring the outbound provisioning
+
+Now we need to enable sending out provisioning requests to the previously configured provisioning system for any user 
+management operation done via the management console, SOAP API or the SCIM API. To do that, you must configure outbound 
+provisioning identity providers against the resident service provider. So, based on the outbound configuration, 
+users created in WSO2 Identity Server can also be provisioned to external systems like HubSpot or Google Apps.
+
+1. Log on to the [Management Console](https://localhost:9443/carbon/) using the username and password.
+2. In the **Main** menu under the **Identity** section, click **Resident** under **Service Providers.**
+3. In the resulting screen, expand the **Outbound Provisioning Configuration** section.
+4. In the **Outbound Provisioning Configuration** section, do the following.
+
+    1. Select the identity provider you added from the drop-down menu available and click the + sign to add it. If you 
+    have not added an identity provider yet, this step is not possible.
+    2. Once added, the identity provider is displayed as an entry in the list. Select scim from the drop-down to ensure 
+    that the SCIM operation is used for provisioning.
+    3. If the option, Blocking is enabled, the outbound provisioning request will be blocked until the response is 
```

Comment:
Boldify the following: 
**Blocking**
**Enable Rules**
**Update**

Reply:
Fixed d61920a